### PR TITLE
Heroku test

### PR DESCRIPTION
**Pull request by prashantpawar**
This branch has the code which allows us to do CI

### CODE REVIEW

**Comment by jimitpatel9** [comment]
:+1:

**Comment by jimitpatel9** [comment]
:+1:

**Comment by jackofseattle** [comment]
LGTM

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+worker: npm run dev
```
Comment (jackofseattle) [inline Procfile]: Shouldn't run dev - it's a production environment

Comment (prashantpawar) [inline Procfile]: Gotcha, good catch, will fix this.

**Pull request merged**